src/zenml/integrations/kubeflow/container_entrypoint.py

```diff
@@ -203,7 +203,7 @@ def _dump_input_populated_artifacts(
             )
             # There must be at least a channel in a input, and all channels in
             # a input share the same artifact type.
-            artifact_type = spec.channels[0].artifact_query.type.name  # type: ignore[attr-defined] # noqa
+            artifact_type = spec.channels[0].artifact_query.type.entrypoint_name  # type: ignore[attr-defined] # noqa
```

Comment:
I think all three changes in this kubeflow entrypoint don't use the post-execution step class and therefore should stay `name`.

Reply:
removed